### PR TITLE
xDAI: Remove burner connect , Add Tor.us

### PR DESCRIPTION
**Pull request by Theylon**
xDAI:
- Remove burner connect (Not in use)
- add Tor.us
- use web3Modal 1.6.3

### CODE REVIEW

File: src/settings.ts [hunk 1]
```diff
@@ -110,13 +110,10 @@ function getWeb3ConnectProviderOptions(network: string) {
...
-          options: {
-            defaultNetwork: "100",
-            defaultWallets: [
-              { origin: "https://buffidao.com/", name: "BuffiDAO" },
-              { origin: "https://judge.buffidao.com/", name: "Judges BuffiDAO Wallet" },
+        torus: {
+          package: Torus,
+          options: { network: "xdai" },
+        },
             ],
```
Comment (orenyodfat) [inline src/settings.ts:117]: remove ],

File: src/settings.ts [hunk 2]
```diff
@@ -110,13 +110,10 @@ function getWeb3ConnectProviderOptions(network: string) {
...
           },
```
Comment (orenyodfat) [inline src/settings.ts:118]: remove },

File: src/settings.ts [hunk 3]
```diff
@@ -110,13 +110,10 @@ function getWeb3ConnectProviderOptions(network: string) {
...
         },
```
Comment (orenyodfat) [inline src/settings.ts:119]: remove },

**Comment by orenyodfat** [comment]
still lint errors

**Comment by Theylon** [comment]
Hey @orenyodfat , it seems like Alchemy does not support ChainID 100. I'll make a PR to remove this optional parameter.

image

**Comment by orenyodfat** [comment]
please submit an issue , not a pr .  and describe what you are seeing and how to reproduce.
the chain id might be needed.

**Pull request merged**